cloudbridge/providers/azure/azure_client.py

```diff
@@ -367,47 +368,47 @@ def list_vm_firewall(self):
 
     def create_vm_firewall(self, name, parameters):
         return self.network_management_client.network_security_groups. \
-            create_or_update(self.resource_group, name,
-                             parameters).result()
+            begin_create_or_update(self.resource_group, name,
+                                   parameters).result()
 
     def update_vm_firewall_tags(self, fw_id, tags):
         url_params = azure_helpers.parse_url(VM_FIREWALL_RESOURCE_ID,
                                              fw_id)
-        name = url_params.get(VM_FIREWALL_NAME)
+        name = url_params.get(VM_FIREWALL_NAME, "")
```

Comment:
This is one of the many "make typing happy" kind of change, let me know if you want me to revert this

Reply:
I think it's fine, thanks for fixing it.